pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.depproj

```diff
@@ -35,6 +39,9 @@
         <TargetPath Condition="'%(_FilesToPackage.IsNative)' != 'true'">runtimes/$(NuGetRuntimeIdentifier)/lib/$(PackageTargetFramework)</TargetPath>
         <TargetPath Condition="'%(_FilesToPackage.IsNative)' == 'true'">runtimes/$(NuGetRuntimeIdentifier)/native</TargetPath>
       </_FilesToPackage>
+      <_FileToPackage Include="$(_runtimePackagePath)/tools/*.*">
```

Comment:
Crossgen has a different name on different OSes and didn't want to special case it.  Also if you decide to bundle more stuff there (like symbols, or other tools) we'd probably want to pick it up by default.